app/components/RandomQuote/RandomQuote.css

```diff
@@ -13,9 +13,8 @@
 }
 
 .quoteText {
-  font-weight: 600;
```

Comment:
@ollfkaih I reduced the `font-weight` like you suggested in #3487

Reply:
This is definitely better. I think the poll and quote components still look like they're not designed by the same people, but this improves the situation.